packages/quantic/cypress/integration/refine-modal-content/refine-modal-content-selectors.ts

```diff
@@ -0,0 +1,96 @@
+import {ComponentSelector, CypressSelector} from '../common-selectors';
+
+const refineContentComponent = 'c-quantic-refine-modal-content';
+const refineContentContainer = '.facets-container';
+
+export interface RefineContentSelector extends FacetSelector {
+  container: FacetSelector;
+  sort: () => CypressSelector;
+  clearAllFiltersButton: () => CypressSelector;
+  filtersTitle: () => CypressSelector;
+}
+
+export interface FacetSelector extends ComponentSelector {
+  get: () => CypressSelector;
+  facetManager: () => CypressSelector;
+  facet: () => CypressSelector;
+  numericFacet: () => CypressSelector;
+  categoryFacet: () => CypressSelector;
+  timeframeFacet: () => CypressSelector;
+  facetManagerItems: () => CypressSelector;
+  timeframeFacetExpandButton: () => CypressSelector;
+  facetExpandButton: () => CypressSelector;
+  timeframeFacetClearFiltersButton: () => CypressSelector;
+  facetClearFiltersButton: () => CypressSelector;
+  timeframeFacetFirstOption: () => CypressSelector;
+  facetFirstOption: () => CypressSelector;
+}
+
+const getCommonFacetSelectors = (container: boolean): FacetSelector => {
+  return {
+    get: () =>
+      cy.get(container ? refineContentContainer : refineContentComponent),
```

Comment:
You could get rid of the ternary on each selector by passing a Cypress selector to `getCommonFacetSelectors` and chaining the other calls from there.

For example:
```
const getCommonFacetSelectors = (rootSelector): FacetSelector => {
  return {
    get: () => rootSelector(),
    facet: () => rootSelector().find('c-quantic-facet'),
    ...
  };
}
```

and you would invoke `getCommonFacetSelectors` this way:
```
getCommonFacetContainer(() => cy.get(refineContentComponent));
getCommonFacetContainer(() => cy.get(refineContentContainer));
```